benchmark/index.js

```diff
@@ -13,9 +13,7 @@ const caps = require('ssb-caps')
 const ssbKeys = require('ssb-keys')
 const multicb = require('multicb')
 const pull = require('pull-stream')
-const asyncFilter = require('pull-async-filter')
```

Comment:
it seems like we can remove this dep now as well?

Reply:
Will do!